<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registrarse</class>
 <widget class="QDialog" name="registrarse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrarse</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Iconos/Registrarse.png</normaloff>../Iconos/Registrarse.png</iconset>
  </property>
  <widget class="QLineEdit" name="lineEdit_1Nombre">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9NumeroDeCelular">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6NumeroDeCedula">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Numero de Cedula</string>
   </property>
  </widget>
  <widget class="Line" name="line_12">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_18">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7DireccionDeResidencia">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Direccion de Residencia</string>
   </property>
  </widget>
  <widget class="Line" name="line_17">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2Apellido">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Apellido</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9NumeroDeCelular">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Numero de Celular</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3FechaDeNacimiento">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fecha de Nacimiento</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1Nombre">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5EstadoCivil">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1Siguiente">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Siguiente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4Sexo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sexo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10CorreoElectronico">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7DirecciondeResidencia">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2Apellido">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6NumeroDeCedula">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4Sexo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8TarjetaDeCredito">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_16">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8TarjetaDeCredito">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tarjeta de Credito (Opcional)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2Cancelar">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5EstadoCivil">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Estado Civil</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3FechaDeNacimiento">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10CorreoElectronico">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Correo Electronico</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_1Nombre</tabstop>
  <tabstop>lineEdit_2Apellido</tabstop>
  <tabstop>lineEdit_3FechaDeNacimiento</tabstop>
  <tabstop>lineEdit_4Sexo</tabstop>
  <tabstop>lineEdit_5EstadoCivil</tabstop>
  <tabstop>lineEdit_6NumeroDeCedula</tabstop>
  <tabstop>lineEdit_7DirecciondeResidencia</tabstop>
  <tabstop>lineEdit_8TarjetaDeCredito</tabstop>
  <tabstop>lineEdit_9NumeroDeCelular</tabstop>
  <tabstop>lineEdit_10CorreoElectronico</tabstop>
  <tabstop>pushButton_1Siguiente</tabstop>
  <tabstop>pushButton_2Cancelar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2Cancelar</sender>
   <signal>clicked()</signal>
   <receiver>registrarse</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_1Siguiente</sender>
   <signal>clicked()</signal>
   <receiver>registrarse</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
